src/studio/src/designer/backend/Services/Implementation/SourceControlSI.cs

```diff
@@ -553,6 +530,44 @@ public void VerifyCloneExists(string org, string repository)
             }
         }
 
+        private void CommitAndPushToBranch(string org, string repository, string branchName, string localPath, string message)
```

Comment:
Consider splitting this into two methods Commit & Push. Makes it a bit more readable and easier to reuse individually.

Reply:
We already have seperate methods for this. We could probably refactor to reuse these methods where commit&Push is used elsewhere in the solution, but that would require a higher test coverage first. Creating a seperate issue for this.